lua/ulx/modules/sh/cfc_brazil.lua

```diff
@@ -0,0 +1,141 @@
+local CATEGORY_TYPE = "Teleport"
+local MIN_X, MIN_Y, MIN_Z = -16384, -16384, -16384
+local MAX_X, MAX_Y, MAX_Z = 16384, 16384, 16384
```

Comment:
how were these numbers obtained?

Reply:
That's how wiremod limits its setPos and so forth.